src/components/mapPageComponents/logic/event.ts

```diff
@@ -156,7 +156,7 @@ export function addHandleZoomChange(
       zoomedIn = true;
     }
     setNeighborhoodsStyle(neighborhoodsLayer, markersArray, !zoomedIn);
-    setPrivateStreetsStyle(privateStreetsLayer, zoomedIn);
+    setPrivateStreetsStyle(privateStreetsLayer, !zoomedIn);
```

Comment:
What this will do is make it so that when a user zooms in, the streets aren't visible, which is good but it'll also make the streets visible when you zoom out. We want to remove visibility at all zoom levels. Changing `!zoomedIn` to `false` would do the trick, or you could also mess with the MapView file directly so that the streets never get loaded into the map at all (if you do this, you'd also have to change this function and anywhere else that sets the private streets style)

Reply:
got it! I think I still want to keep the functionality to have the option to load the streets on the map but this is good to know